src/sophios/compiler.py

```diff
@@ -884,7 +884,7 @@ def compile_workflow_once(yaml_tree_ast: YamlTree,
                 if in_dict['value'].startswith('/'):
                     print("Warning! directory can not start with '/'")
                     print("It is most likely an incorrect path! Can't create directories!")
-                    sys.exit(1)
+                    # sys.exit(1)
```

Comment:
removing the code is better than commenting it out :-)

Reply:
I am removing the whole block in PR 284. It was just a testing comment.